src/polymer/behavior-scanner.ts

```diff
@@ -104,7 +104,7 @@ class BehaviorVisitor implements Visitor {
       }
       if (name in this.propertyHandlers) {
         this.propertyHandlers[name](prop.value);
-      } else if (babel.isFunction(prop.value)) {
+      } else if (babel.isMethod(prop) || babel.isFunction(prop.value)) {
```

Comment:
What about when we're scanning legacy polymer element bodies and classes? Are we already checking for shorthand methods there?

Reply:
- `polymer-element-scanner`: Already worked, added a test case in new commit.
- `polymer2-element-scanner`: Already worked, including test case (https://github.com/Polymer/polymer-analyzer/blob/master/src/test/static/polymer2/test-element-10.js).
- `polymer-core-feature-scanner`: Added shorthand method support and test in new commit.